pkg/apis/numaflow/v1alpha1/edge_types.go

```diff
@@ -21,7 +21,7 @@ import "fmt"
 type Edge struct {
 	From string `json:"from" protobuf:"bytes,1,opt,name=from"`
 	To   string `json:"to" protobuf:"bytes,2,opt,name=to"`
-	// Conditional forwarding, only allowed when "From" is a Sink or UDF.
+	// Conditional forwarding, only allowed when "From" is a Source or UDF.
```

Comment:
hmmmm. the `from` can only be a `Source` or `UDF`, right? do we have that validation?

Reply:
oh... this is a doc fix.